InternProject/DAL/Models/OrderTrack/ProgressCheckModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Web;
+
+namespace InternProject.DAL.Models.OrderTrack
+{
+    public class ProgressCheckModel
+    {
+        [Key]
+        [Required]
+        public int ID { get; set; }
+
+        //yes/no
+        public string Complete { get; set; }
+
+        //yes/no
+        public string onSchedule { get; set; }
```

Comment:
naming convention

Reply:
change from "onSchedule" to "OnSchedule"

Reply:
change from "string" to "bool"